dependencies/common/install-common

```diff
@@ -30,9 +30,15 @@ set -e
 # via sudo; however, this causes problems if used when building a Dockerfile
 if [ "$1" != "--install-crashpad-sudo" ]
 then
-   ./install-crashpad
+  if [[ "$OSTYPE" == "darwin"* ]]; then 
+    # use pre-built binaries on macos as compilation environment is difficult
+    # to configure there
+    ./install-crashpad osx
+  else 
+    ./install-crashpad
```

Comment:
Ideally we should do this for every Linux platform as well, but the autodetection is difficult to do there. These are forwarded specifically in the docker containers.